src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -1292,4 +1293,1368 @@ void EIRPlantLoopHeatPump::oneTimeInit(EnergyPlusData &state)
+                        this->loadSideOutletTemp = state.dataLoopNodes->Node(this->loadSideNodes.outlet).TempSetPoint;
+                    } break;
+                    case DataPlant::LoopDemandCalcScheme::DualSetPointDeadBand: {
+                        this->loadSideOutletTemp = state.dataLoopNodes->Node(this->loadSideNodes.outlet).TempSetPointHi;
+                    } break;
+                    default:
+                        break;
+                    }
+                } else {
+                    // Try to request zero flow
+                    this->loadSideMassFlowRate = 0.0;
+                    // Use PlantUtilities::SetComponentFlowRate to decide actual flow
+                    PlantUtilities::SetComponentFlowRate(
+                        state, this->loadSideMassFlowRate, this->loadSideNodes.inlet, this->loadSideNodes.outlet, this->loadSidePlantLoc);
+                    // No deltaT since component is not running
+                    this->loadSideOutletTemp = state.dataLoopNodes->Node(this->loadSideNodes.inlet).Temp;
+                    // this->QEvaporator = 0.0;
+                    // PartLoadRat = 0.0;
+                    // this->ChillerPartLoadRatio = 0.0;
+
+                    // if (this->DeltaTErrCount < 1 && !state.dataGlobal->WarmupFlag) {
+                    if (!state.dataGlobal->WarmupFlag) {
+                        // ++this->DeltaTErrCount;
+                        ShowWarningError(state, "FFHP evaporator DeltaTemp = 0 in mass flow calculation (Tevapin = Tevapout setpoint temp).");
+                        ShowContinueErrorTimeStamp(state, "");
+                        // } else if (!state.dataGlobal->WarmupFlag) {
+                        // ++this->ChillerCapFTError;
+                        ShowWarningError( // RecurringWarningErrorAtEnd(
+                            state,
+                            format("{} \"{}\": FFHP evaporator DeltaTemp = 0 in mass flow calculation warning continues...",
+                                   DataPlant::PlantEquipTypeNames[static_cast<int>(this->EIRHPType)],
+                                   this->name));
+                        // this->DeltaTErrCountIndex,
+                        // evapDeltaTemp,
+                        // evapDeltaTemp);
+                    }
+                }
+            }
+        } else { // If FlowLock is True
+            this->loadSideMassFlowRate = state.dataLoopNodes->Node(this->loadSideNodes.inlet).MassFlowRate;
+            PlantUtilities::SetComponentFlowRate(
+                state, this->loadSideMassFlowRate, this->loadSideNodes.inlet, this->loadSideNodes.outlet, this->loadSidePlantLoc);
+            //       Some other component set the flow to 0. No reason to continue with calculations.
+            if (this->loadSideMassFlowRate == 0.0) {
+                FFHPloadSideLoad = 0.0;
+                // return;
+            }
+        } // This is the end of the FlowLock Block
+    } else {
```

Comment:
Are these disembodied `else`'s here for a reason?

Reply:
Removed in earlier commits.